cobra/flux_analysis/loopless.py

```diff
@@ -82,6 +81,26 @@ def add_loopless(model, zero_cutoff=1e-12):
         model.constraints[name].set_linear_coefficients(coefs)
 
 
+def __add_cycle_free(model, fluxes):
```

Comment:
Not really. Hoped that this would obfuscate the name as for class members so it wont be used accidentally by the user. will change it to one.